Laura/index.js

```diff
@@ -0,0 +1,22 @@
+// # 12 - PB - data - types - pt2
+
+// ** Strings, Numbers, Booleans **
+
+// 1. Declare a variable and assign a string as a value. Make sure the string is at least 6 characters long.Print the variable to the console.
+let str = "hello world";
+console.log(str); // hello world
+
+// 2. Access the 2nd and 6th elements of the string.Print the elements to the console. ** Make sure you have selected the right index numbers **
+console.log(str[2], str[6]); // l w
```

Comment:
The index of second element is 1 and index of 6th element is 5.
console.log(str[1], str[5]);